fuel/app/classes/materia/session/playdataexporter.php

```diff
@@ -376,4 +376,62 @@ protected static function questions_and_answers($inst, $semesters)
 
 		return [$string, '_questions_answers.csv'];
 	}
+
+	// Outputs a .zip file of two CSV files for individual and collective referrers data
+	protected static function referrer_links($inst, $semesters)
```

Comment:
Change the export dropdown to read `Referrer Urls`

Reply:
Done. Now appears as `Referrer URLs`